python/cugraph/__init__.py

```diff
@@ -21,6 +21,7 @@
     subgraph,
     triangles
 )
+from cugraph.centrality import katz_centrality
```

Comment:
Just to double check, in an older email thread, we discussed to expose the underlying framework to python users (e.g. cugraph.thirdparty.hornet.katz_centrality instead of cugraph.katz_centrality), but have we changed this to hide the underlying framework to users?

Reply:
cuHornet is not a thirdparty library,  so it should be cugraph.cuhornet.katz_centrality.  
BUT ...   we also need a cugraph.katz_centrality function.  That top level call should pick the best sub-solution, which in this case is just the cuhornet version

Reply:
Agreed. @kaatish proposal is pretty user-friendly based on what we have. As a user what I would really like to do is `from cugraph.centrality import katz_centrality` and I don't need to be aware of the backend details especially if there's a single backend.